core/arch/arm/plat-zynqmp/main.c

```diff
@@ -57,6 +57,9 @@ register_phys_mem_pgdir(MEM_AREA_IO_SEC,
 register_phys_mem_pgdir(MEM_AREA_IO_SEC,
 			ROUNDDOWN(GIC_BASE + GICD_OFFSET, CORE_MMU_PGDIR_SIZE),
 			CORE_MMU_PGDIR_SIZE);
+#if defined(CFG_CSU)
```

Comment:
`CFG_CSU` is already used by plat-imx. Perhaps `CFG_ZYNQMP_CSU` would make sense?

Reply:
works for me